src/shared/src/components/LandmarkShortcuts.tsx

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import cn from 'classnames';
+
+export interface ILandmarkShortcutsProps {
+  shortcuts: ILandmarkShortcut[];
+}
+
+interface ILandmarkShortcut {
+  id: string;
+  text: string;
+}
+
+const useStyles = makeStyles({
+  button: {
+    textDecoration: 'underline',
+    textAlign: 'left',
+    padding: '5px',
+    paddingLeft: '10px',
+  },
+  jumpNav: {
+    backgroundColor: "#fff",
+  },
+  srOnly: {
+    position: 'absolute',
+    clip: 'rect(0,0,0,0)',
+    border: 0,
```

Comment:
Is removing all the other attributes "sr-only enough" for all supported browsers?

Reply:
Tested OK on safari + chrome. Do you have any reason to believe this is not gonna work?

Suggest we test this. Dragged to test in the board.